### DISCUSSION TITLE
doris聚合表副本不一致，不一致的tablet太多，手动修复困难，求解~

### DISCUSSION THREAD

**Discussion by Peiking** [apache/doris#13291] [A - General / Q&A]
执行查看结果 ADMIN SHOW REPLICA STATUS FROM db.table PARTITION (p20220607) ;
status都是OK的，但是同一个tablet的不同副本的VersionNum不一致

执行 SHOW PROC '/dbs/2550812/3192404/partitions/3192403/3194396/3194477';
且RowCount-数据条数也不一致。

ADMIN REPAIR TABLE 无法修复status都是OK的副本。

有问题的tablet很多有100多个，手动修复太多了，求解~

**Comment by zy-kkk** [comment]
In this case, you can choose to set the wrong tablet state to bad. This can only be done manually.
Or you can just delete the partition and repopulate it from upstream
